packages/opentelemetry-exporter-collector/src/types.ts

```diff
@@ -341,6 +343,15 @@ export interface CollectorExporterConfigBase {
   url?: string;
 }
 
+/**
+ * Collector Exporter node base config
+ */
+export interface CollectorExporterNodeConfigBase
```

Comment:
this should be defined in new file `types.ts` in `packages/opentelemetry-exporter-collector/src/platform/node`
because `CollectorExporterNodeConfigBase` is not needed for grpc and for web
remember to add export for this file in
`packages/opentelemetry-exporter-collector/src/platform/node/index.ts`